packages/components/badge/stories/badge.stories.tsx

```diff
@@ -68,7 +68,7 @@ const defaultProps = {
 const Template = (args: BadgeProps) => (
   <Badge {...args}>
     <Avatar
-      isBordered={args.classNames?.badge?.includes("bottom")}
+      isBordered={(args.classNames?.badge as string)?.includes("bottom")}
```

Comment:
`args.classNames?.badge` could be something like `["flex", "z-10", ...]`

Reply:
Hey @winchesHe yes it can also be an array which could make the ".includes" validation fail, please double check

Reply:
fixed